lib/frozen_record/base.rb

```diff
@@ -64,7 +68,7 @@ def current_scope=(scope)
 
       def file_path
         raise ArgumentError, "You must define `#{name}.base_path`" unless base_path
-        File.join(base_path, "#{name.underscore.pluralize}.yml")
+        File.join(base_path, "#{name.underscore.pluralize}.#{backend.extension}")
```

Comment:
I think you should pass `base_path` and `name` to the backend, and let it decide what the naming convention is.

Reply:
That's a great idea, it even allow things like backends specifying custom folders inside `base_path`. 👍

Reply:
On second though, I would simply pass the model as a single argument.

The backend can then get the `base_path` etc from there.